README.rst

```diff
@@ -0,0 +1,219 @@
+develop applications keeping in mind they are to run in a cluster environment.
+
+As a **cluster management tool**, Tarantool Cartridge provides your cluster-aware
+applications with the following key benefits:
+
+* horizontal scalability and load balancing via built-in automatic sharding;
+* asynchronous replication;
+* automatic failover;
+* centralized cluster control via GUI or API;
+* automatic configuration synchronization;
+* instance functionality segregation.
+
+A Tarantool Cartridge cluster can segregate functionality between instances via
+built-in and custom (user-defined) **cluster roles**. You can toggle instances
+on and off on the fly during cluster operation. This allows you to put
+different types of workloads (e.g., compute- and transaction-intensive ones) on
+different physical servers with dedicated hardware.
+
+Tarantool Cartridge has an external utility called
+`cartridge-cli <https://github.com/tarantool/cartridge-cli>`_ which
+provides you with utilities and an application template to help:
+
+* easily set up a development environment for your applications;
+* plug the necessary Lua modules;
+* pack the applications in an environment-independent way: together with
+  module binaries and Tarantool executables.
+
+--------------------------------------------------------------------------------
+Getting started
+--------------------------------------------------------------------------------
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Create your first application
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To get a template application that uses Tarantool Cartridge and run it,
+you need to install the `cartridge-cli` utility (supposing that
+`Tarantool <https://www.tarantool.io/en/download/>`_ is already installed).
+
+Long story short, copy-paste this into the console:
+
+.. code-block:: console
+
+    tarantoolctl rocks install cartridge-cli
+    .rocks/bin/cartridge create --name myapp
+    cd myapp
+    ../.rocks/bin/cartridge build
+    ../.rocks/bin/cartridge start
+
+That's all! Now you can visit http://localhost:8081 and see your application's
+Admin Web UI:
+
+.. image:: https://user-images.githubusercontent.com/11336358/75786427-52820c00-5d76-11ea-93a4-309623bda70f.png
+   :align: center
+   :scale: 100%
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Next steps
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+See:
+
+* A more detailed
+  `getting started guide <https://www.tarantool.io/en/doc/2.2/getting_started/getting_started_cartridge/>`_
+* More
+  `application examples <https://github.com/tarantool/examples>`_
+* `Cartridge documentation <https://www.tarantool.io/en/doc/latest/book/cartridge/>`_
+* `Cartridge API reference <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_api/>`_
```

Comment:
These links are ok for github readme, but they look weird inside book itself. What can we do about them?

Reply:
We can't use local label names to refer to tarantool/doc files if we want links from this project (from readme and from other topics of the cartridge doc) to work outside of our tarantool.io assembly.

Reply:
But we can try to link the projects via intersphinx plugin.